Client/Client.Qt/Widgets/MessageWidget.cpp

```diff
@@ -18,12 +18,12 @@ MessageWidget::MessageWidget(QWidget* history, QString message, uint64_t userId,
       _messageId(messageId),
       _messageText(std::move(message)),
       _username(std::move(username)),
-      _datetime(QDateTime::fromMSecsSinceEpoch(utc)),
+      _datetime(QDateTime::fromSecsSinceEpoch(utc)),
```

Comment:
Try to use msecs since using secs might create problem in future

Reply:
Could you elaborate pls?